_data/glossary.yml

```diff
@@ -15,3 +15,15 @@ element-instance-resource: "The resources available to the element instance thro
 resource: "Specific endpoints within an API that contain objects accessible via the API. "
 
 one-to-many: "Enables developers to write to one API to integrate multiple elements."
+
+formula: "A reusable workflow that is independent of the API provider, or Element."
+
+formula-template: "A template for a formula that includes the triggers, steps, and variables for a formula instance to execute a workflow. "
+
+formula-instance: "A specific instance of a formula template configured with explicit variables and associated with specific element instances."
+
+formula-trigger: "An event that occurs and kicks off a formula. Triggers can be events set up on an element instance, an API call to an element instance, a scheduled occurrence, or manually triggered."
```

Comment:
formula-template: "A reusable workflow that is independent of the Element and includes the triggers, steps, and variables for a formula instance to execute the workflow. "

formula-instance: "A specific instance of a formula template configured with explicit variables and associated with specific element instances."